### PR TITLE
134249295/czar to draw cards by clicking

### PR DESCRIPTION
**Pull request by raphaelakpan**
#### What does this PR do?
The Czar select the next set of cards by clicking on a deck of cards.

#### Description of Task to be completed?
This tasks enables the Czar to select the next set of cards by clicking on a deck of cards.

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
Previously, after the Czar selects the winner, another set of cards are randomly selected and the next round begins, rather leaving the czar seemingly confused about what just happened.

#### What are the relevant pivotal tracker stories?
#134249295
#### Screenshots (if appropriate)
* Czar

image

* Other Players

image

#### Questions:
N/A

### CODE REVIEW

File: config/socket/game.js [hunk 1]
```diff
@@ -136,13 +140,22 @@ Game.prototype.startGame = function() {
   console.log(this.gameID,this.state);
   this.shuffleCards(this.questions);
   this.shuffleCards(this.answers);
+  // this.stateDrawCards(this);
```
Comment (andela-aolaniran) [inline config/socket/game.js]: Commented code should be removed

File: config/socket/game.js [hunk 2]
```diff
@@ -158,7 +171,7 @@ Game.prototype.stateChoosing = function(self) {
   }
   self.round++;
   self.dealAnswers();
-  // Rotate card czar
+  // // Rotate card czar
```
Comment (andela-aolaniran) [inline config/socket/game.js]: repeated single line comments

File: config/socket/game.js [hunk 3]
```diff
@@ -179,9 +192,11 @@ Game.prototype.selectFirst = function() {
     this.players[winnerIndex].points++;
     this.winnerAutopicked = true;
     this.stateResults(this);
+    this.sendNotification(this.players[winnerIndex].username+' has won the round!');
+    this.sendUpdate();
   } else {
     // console.log(this.gameID,'no cards were picked!');
```
Comment (andela-aolaniran) [inline config/socket/game.js]: Remove console.log statements

File: public/js/controllers/game.js [hunk 4]
```diff
@@ -1,25 +1,31 @@
...
     $scope.showTable = false;
     $scope.modalShown = false;
     $scope.game = game;
     $scope.pickedCards = [];
-    var makeAWishFacts = MakeAWishFactsService.getMakeAWishFacts();
+    let makeAWishFacts = MakeAWishFactsService.getMakeAWishFacts();
     $scope.makeAWishFact = makeAWishFacts.pop();
     $scope.chat = game.gameChat;
-
+    $scope.userName = Storage.get('user');
```
Comment (andela-aolaniran) [inline public/js/controllers/game.js]: Cool.

File: config/socket/game.js (hunk 1 above)
Comment (raphaelakpan) [reply config/socket/game.js]: Done...

File: config/socket/game.js (hunk 2 above)
Comment (raphaelakpan) [reply config/socket/game.js]: Done...

File: config/socket/game.js (hunk 3 above)
Comment (raphaelakpan) [reply config/socket/game.js]: I'll remove the ones that are commented out. 

Some of are actually useful.

Done

**Review by neddinn — APPROVED** [review]

File: config/socket/socket.js [hunk 5]
```diff
@@ -99,6 +99,12 @@ module.exports = function(io) {
       connectedUsers[socket.userEmail] = undefined;
       exitGame(socket);
     });
+
+    socket.on('drawCard', function() {
```
[the remark this answers is not in the document]
Comment (andela-oakinseye) [reply config/socket/socket.js:103]: Is it possible to use an arrow function on this line?

**Pull request merged**